minibear/src/main/java/frc/robot/commands/IntakePonderCommand.java

```diff
@@ -0,0 +1,91 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: DefaultEveryBotPickerCommand.java
+// Intent: Forms a command to uptake/expel the cargo from the every bot picker.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.common.WristPosition;
+import frc.robot.subsystems.IntakeSubsystem;
+import frc.robot.subsystems.WristSubsystem;
+
+import java.util.function.DoubleSupplier;
+
+public class IntakePonderCommand extends CommandBase {
```

Comment:
this is mostly duplicated code from IntakeDefaultCommand. Suggest to combine them and have a mode enum that switches between ponder and normal modes.